plugins/backend/backstage-plugin-argo-cd-backend/README.md

```diff
@@ -16,16 +16,27 @@ argocd:
       instances:
         - name: argoInstance1
           url: https://argoInstance1.com
-          token: ${ARGOCD_AUTH_TOKEN}
+          token: ${ARGOCD_AUTH_TOKEN} # optional 
         - name: argoInstance2
           url: https://argoInstance2.com
+          # dedicated username/password for this instance
+          username: ${ARGOCD_USERNAME_INSTANCE_2} # optional 
+          password: ${ARGOCD_PASSWORD_INSTANCE_2} # optional 
 ```
 
-Add the required auth tokens to environmental variables, `ARGOCD_USERNAME` and `ARGOCD_PASSWORD`.
+### Authentication
+
+- Option 1: Add the required auth tokens to environmental variables, `ARGOCD_USERNAME` and `ARGOCD_PASSWORD` inside the `argocd` object. It will be use as credentials for all instances by default.
```

Comment:
What should I set the `instances` config too if I am using this option.. I would guess the following, am I correct?

```
        - name: argoInstance1
          url: https://argoInstance1.com
```

Reply:
I would suggest adding the example config under each option.